docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+# docker-compose.yml
+# Local + Prod via profiles. Use: 
+#   Local: docker compose --profile local up -d
+#   Prod:  docker compose --profile prod up -d
+name: essosc-viewer
+
+volumes:
+  geoserver_data_dir:
+  traefik_letsencrypt:
+
+networks:
+  internal:
+  web: # shared with reverse proxy (prod)
```

Comment:
make a better name for 'web' because 'internal' is also 'internal' and 'web' is something internal?

Reply:
fixed in 5b079b8